homeassistant/components/unifi/config_flow.py

```diff
@@ -217,6 +219,23 @@ def __init__(self, config_entry):
             ),
         )
 
+    async def async_step_device_tracker_ssid_filter(self, user_input=None):
+        """Manage the device tracker ssid filter option."""
+        if user_input is not None:
+            self.options.update(user_input)
+            return await self.async_step_statistics_sensors()
+
+        controller = get_controller_from_config_entry(self.hass, self.config_entry)
+
+        ssid_filter = {wlan: wlan for wlan in controller.api.wlans}
+
+        return self.async_show_form(
+            step_id="device_tracker_ssid_filter",
+            data_schema=vol.Schema(
+                {vol.Optional(CONF_SSID_FILTER): cv.multi_select(ssid_filter)}
```

Comment:
You should add a default with the current values

Reply:
Done 👍